bots/tests-invoke

```diff
@@ -149,18 +155,12 @@ class PullTask(object):
     def rebase(self, offline=False):
         origin_base = "origin/" + self.base
 
-        # Retrieve information about our base branch
-        if not offline:
-            subprocess.check_call([ "git", "fetch", "origin", self.base ])
-        if self.sink:
-            commit = subprocess.check_output([ "git", "rev-parse", origin_base ]).strip()
-            self.sink.status["master"] = commit
-
         # Rebase this branch onto the base, but only if it's not already an ancestor
         try:
-            if subprocess.call([ "git", "merge-base", "--is-ancestor", origin_base "HEAD" ]) != 0:
+            if subprocess.call([ "git", "merge-base", "--is-ancestor", origin_base, "HEAD" ]) != 0:
```

Comment:
That should be in the previous commit too. Not sure how much we care about bisectability for this, but if it's not too much trouble it would be good to not have known broken commits in master.